test/test_transformers.py

```diff
@@ -3302,6 +3302,8 @@ def _get_mem_eff_drop_mask(batch_size, n_heads, q_len, kv_len, p, seed, offset,
                 fudge_factors['grad_query'] = 650.0
             if dtype == torch.float32:
                 fudge_factors['grad_key'] = 90.0
+                if "gfx95" in torch.cuda.get_device_properties(0).gcnArchName:
+                    fudge_factors['grad_value'] = 12.0
```

Comment:
@dnikolaev-amd @xinyazhang Is this upstreamed already? Will we need this for MI355 transformers tests to pass on upstream, or is it only needed for aotriton 0.9.2b?